### PR TITLE
Sidebar Animation

### PR DESCRIPTION
**Pull request by G1TE5H**
Issue #3149 

Previously there was no animation for sidebars
Like this
j no anim

Fix for this
j anim

EDIT: 
Animation is smooth in real but in gif it looks like they lag.

@PackElend Please Label This

**Changes**
In MainScreen.jsx

### CODE REVIEW

File: ElectronClient/gui/MainScreen.jsx [hunk 1]
```diff
@@ -495,23 +495,23 @@ class MainScreenComponent extends React.Component {
 			height: rowHeight,
 			display: 'inline-block',
 			verticalAlign: 'top',
+			transition: 'all 0.15s ease',
 		};
 
 		if (isSidebarVisible === false) {
 			this.styles_.sideBar.width = 0;
-			this.styles_.sideBar.display = 'none';
```
Comment (G1TE5H) [inline ElectronClient/gui/MainScreen.jsx:502]: Removing this line is important for making transition(animation) working

File: ElectronClient/gui/MainScreen.jsx [hunk 2]
```diff
@@ -495,23 +495,23 @@ class MainScreenComponent extends React.Component {
...
 			width: noteListWidth - this.styles_.verticalResizer.width,
 			height: rowHeight,
 			display: 'inline-block',
 			verticalAlign: 'top',
+			transition: 'all 0.15s ease',
 		};
 
 		if (isNoteListVisible === false) {
 			this.styles_.noteList.width = 0;
-			this.styles_.noteList.display = 'none';
```
Comment (G1TE5H) [inline ElectronClient/gui/MainScreen.jsx:514]: Removing this line is important for making transition(animation) working

**Comment by Rahulm2310** [comment]
@G1TE5H Why ReactNativeClient/package.json & ReactNativeClient/package-lock.json files are also modified ? Please remove those extra changes

**Comment by G1TE5H** [comment]
@Rahulm2310 I didn't did it intentionally.
I think thats why I am having this error `All check have failed `, Isn't it?

**Pull request closed without merging**